### DISCUSSION TITLE
Where is the folder ./admob_api/ and the file MobileAds.tscn?

### DISCUSSION THREAD

**Discussion by littset50** [Poing-Studios/Godot-AdMob-Android-iOS#116] [Q&A]
The Wiki   [https://github.com/Poing-Studios/Godot-admob-android/wiki/AdMob-API](url)  says:

> Go to main page of this repository
> Downloading the .ZIP
> Finishing the Installation: Copying the ./admob_api/ to your Game Project
> Inside your Project Settings add the res://admob_api/MobileAds.tscn on AutoLoad as Singleton

But I can not find the folder ./admob_api/ and the file **MobileAds.tscn** in the main page  [https://github.com/Poing-Studios/godot-admob-android](url)   or in the zip file.

<img width="667" alt="1" src="https://user-images.githubusercontent.com/7975789/171586354-9e84c252-fb5e-466e-829f-e2e2a357752f.png">

Can anyone give me some helps? Thanks.

**Comment by gumaciel** [comment]
Hi, sorry for this but the documentation is outdated, please check how use our plugin by watching the video tutorial: https://youtu.be/MW4hUR0d9lc
